package.json

```
@@ -22,8 +22,10 @@
   ],
   "scripts": {
     "start": "npm run watch",
-    "build": "node scripts && npm run tsc.prod && npm run rollup.prod.ci",
-    "build.prod": "node scripts && npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
+    "clean": "rm -rf build/ cli/ compiler/ dev-server/ internal/ mock-doc/ sys/ testing/ && npm run clean-scripts",
+    "clean-scripts": "rm -rf sys-node-bundle-cache",
```

Should this be `scripts/build/sys-node-bundle-cache`? That's the only folder with that name I'm able to find:
```
$ find . -name sys-node-bundle-cache
./scripts/build/sys-node-bundle-cache
```
```suggestion
    "clean-scripts": "rm -rf scripts/build/sys-node-bundle-cache",
```